packages/themes/src/g90.js

```diff
@@ -58,6 +58,7 @@ export const text02 = gray30;
 export const text03 = gray60;
 export const text04 = white;
 export const text05 = gray50;
+export const textError = red30;
```

Comment:
I think `red30` and `red40` will need to be added above, similar to that note above!

Reply:
Ok, I knew there was probably something else I needed to do.